azure_pipeline.yaml

```diff
@@ -156,6 +156,22 @@ parameters:
       storage_account_name: 'cftappsdemo'
       dependsOn: 'sbox_frontendappgateway'
 
+    - deployment: 'test_frontendappgateway'
+      environment: 'test'
+      component: 'frontendappgateway'
+      service_connection: 'dcd-cftapps-test'
+      storage_account_rg: 'core-infra-test-rg'
+      storage_account_name: 'cftappstest'
+      dependsOn: 'Precheck'
```

Comment:
for both `dependsOn `is incorrect

Reply:
Have updated to `sbox_frontendappgateway`